roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -0,0 +1,2 @@
+[DEFAULT]
+my_ip = {{ ctlplane_ip }}
```

Comment:
Note to myself: check if nova accepts a hostname/FQDN here, just for curiosity. :)